python-avd/pyavd/_cv/client/async_decorators.py

```diff
@@ -121,59 +121,64 @@ class GRPCRequestHandler:
         max_retries (int): Maximum number of retry attempts for Status.UNAVAILABLE. Total attempts = 1 + max_retries.
         initial_delay (int): Initial delay in seconds before the first retry.
         factor (int): Multiplier for the delay in subsequent retries.
-        list_field (str): Name of the parameter to be split if Status.RESOURCE_EXHAUSTED is received.
+        iter_field (str): Name of the parameter to be split if Status.RESOURCE_EXHAUSTED is received.
```

Comment:
I think `split_field` is a better name.
```suggestion
        split_field (str): Name of the parameter to be split if Status.RESOURCE_EXHAUSTED is received.
```